internal/locale/locale.go

```diff
@@ -34,14 +35,20 @@ func init() {
 	logging.Debug("Init")
 
 	locale := getLocaleFlag()
-	cfg, err := config.Get()
-	if err == nil {
-		setErr := cfg.Set("Locale", "en-US")
-		if err != nil {
-			logging.Error("Could not set locale entry in config, error: %v", setErr)
-		}
-		if locale == "" {
+	if locale == "" {
+		cfg, err := config.New()
```

Comment:
By creating a new instance here, we would also create a new `singlethread`.  I think that also happens in  `auth.go`.